src/simulation/LoadGenerator.h

```diff
@@ -264,6 +266,7 @@ class LoadGenerator
     // Set when load generation actually begins
     std::unique_ptr<VirtualClock::time_point> mStartTime;
 
+    uint32_t mTransactionsAppliedAtTheStart;
```

Comment:
0 initialize this here or in the constructor.

Reply:
initialized to 0 just in case. Note that the correct value is set in `start` since there might be txs applied prior this loadgen run

Reply:
Ya I see that we set it before using it, I just don't like leaving the constructor with uninitialized state.